gpytorch/lazy/lazy_variable.py

```diff
@@ -215,11 +217,41 @@ def size(self):
         """
         raise NotImplementedError
 
+    def transpose(self, dim1, dim2):
+        """
```

Comment:
The implementation below makes me think this was supposed to be in `InterpolatedLazyVariable`, not `LazyVariable` (e.g., it expects `self.left_interp_indices` to exist)

Reply:
Lol I honestly have no idea what I was thinking here!
I think I was hoping to write a generic transpose method that calls the `_transpose_no_batch` method

Reply:
I'll figure out what was going on here.